lib/core/src/lib/dialogs/unsaved-changes-dialog/unsaved-changes.guard.ts

```diff
@@ -48,7 +48,10 @@ export class UnsavedChangesGuard implements CanDeactivate<any> {
                       data: this.data
                   })
                   .afterClosed()
-                  .pipe(tap((confirmed) => (this.unsaved = !confirmed)))
+                  .pipe(
+                      tap((confirmed) => (this.unsaved = !confirmed)),
+                      map((confirmed) => !!confirmed)
```

Comment:
why we need this? I would expect confirmed is boolean (true or false) so !!true gives us just true.